src/semgrep_agent/main.py

```diff
@@ -325,6 +326,7 @@ def protected_main(
             click.secho(message, err=True, fg="red")
             sys.exit(1)
         config = (str(local_config_path),)
+        logging.info(f"Config saved to { SEMGREP_RULES_FILE }")
```

Comment:
```suggestion
        logging.info(f"Config saved to {SEMGREP_RULES_FILE}")
```

I think black might be broken if it's not enforcing this style :/